modules/private-clusters-setting-api-private.adoc

```diff
@@ -37,48 +38,56 @@ ifndef::cpmso-using-azure[]
 *** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
 endif::cpmso-using-azure[]
 ifndef::cpmso-using-aws[]
-*** For Azure, delete the `api-internal` rule for the load balancer.
+*** For Azure, delete the `api-internal-v4` rule for the load balancer.
 endif::cpmso-using-aws[]
 
 .. Delete the `api.$clustername.$yourdomain` DNS entry in the public zone.
```

Comment:
on azure, it is `api.$clustername`, is it worth changing?
<img width="1654" alt="azure" src="https://github.com/openshift/openshift-docs/assets/37139281/ef84a028-8642-451e-996a-79a89bc55eb6">